components/user-settings/UserSettings.tsx

```diff
@@ -0,0 +1,124 @@
+'use client';
+import React, { useEffect, useState } from 'react';
+import {
+  Button,
+  EButtonTypes,
+  EIConTypes,
+  EInputFieldGroupType,
+  EStateInputFieldGroup,
+  ETypographyLevels,
+  Heading,
+  InputFieldGroup,
+} from '@ost-cas-fee-adv-23-24/elbmum-design';
+import useUserInfo from '@/hooks/useUserInfo';
+
+export interface INewUserSettings {
+  firstName: string;
+  lastName: string;
+  email: string;
+}
+
+interface IProps {
+  onClose: () => void;
+  onSave: (options: INewUserSettings) => void;
+}
+const UserSettings = ({ onClose, onSave }: IProps) => {
+  const { firstName, lastName } = useUserInfo();
+  const [firstNameIntern, setFirstNameIntern] = useState('');
+  const [lastNameIntern, setLastNameIntern] = useState('');
+  const [emailIntern, setEmailIntern] = useState('');
+
+  useEffect(() => {
+    setFirstNameIntern(firstName);
+    setLastNameIntern(lastName);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  // todo: Add Name attribute to the input fields (design system)
+  // only after that we could use a form and submit via server actions
+  // in figma there is only one group of data, which includes password.
+  // ¯\_(ツ)_/¯ makes no sense. we need to split it up.
```

Comment:
If I remember correctly, we don't need to implement user settings. Let's ask the teachers and afterwards discuss if this PR is needed

Reply:
There are designs. I would add the component and clearly say that there is no apis on save the data. 
otherwise you have a button in the header with no  purpose.

Reply:
Alright, let's utilize this PR for the UserSettings. However, I suggest modifying the comment to clarify that it's not a forgotten todo but rather a deliberate decision not to implement it for the final submission due to the API functionality